static/scripts/entries.ts

```diff
@@ -0,0 +1,3 @@
+const typescriptEntries = ["static/scripts/app.ts", "static/scripts/audit.ts", "static/scripts/secret.ts", "static/scripts/keygen.ts"];
+const CSSEntries = ["static/styles/app.css", "static/styles/audit.css", "static/styles/sec.css"];
```

Comment:
Wouldn't it be cleaner to dynamically load the files via something like 
```typescript
fs.readDirSync("static/scripts/").filter(name => name.endsWith(".ts"))
```

Reply:
Not necessarily, some files are imported so they don't need to be directly added to the `entryPoints`, reducing the overhead.

Reply:
Perhaps we should make a single entry point then for ts and css making both consistent.

Reply:
single `entryPoints` will bundle unnecessary files